sydent/config/email.py

```diff
@@ -70,6 +70,23 @@ def parse_config(self, cfg: ConfigParser) -> bool:
             "email", "email.third_party_invite_domain_obfuscate_characters"
         )
 
+        third_party_invite_homeserver_blocklist = cfg.get(
+            "email", "email.third_party_invite_homeserver_blocklist", fallback=""
+        )
+        third_party_invite_room_blocklist = cfg.get(
+            "email", "email.third_party_invite_room_blocklist", fallback=""
+        )
+        self.third_party_invite_homeserver_blocklist = {
+            server
+            for server in third_party_invite_homeserver_blocklist.split("\n")
```

Comment:
Can we insert newlines in the config?

Reply:
Yep, like so:
```
some_section.some_list_option =
        item1
        item2
```
I started with comma separated strings, but that looked unwieldy with long lists.

Reply:
Cool!